server/dbtest.js

```diff
@@ -2,16 +2,18 @@ const Pool = require("pg").Pool;
 
```

Comment:
Let's delete this file it's not actually doing anything for the project. It was just to test/demonstrate the db connection.